_dev/apps/ui/src/components/campaign-creation/campaign-creation.vue

```diff
@@ -631,7 +630,7 @@ export default defineComponent({
         ?.currencyCode;
     },
     currencySymbol(): string {
```

Comment:
as now the function can return undefined, the return type should be updated
```typescript
currencySymbol(): string | undefined {
```
Or even better : we could let TS define it for us
```typescript
currencySymbol() {
```